hack/update-codegen.sh

```diff
@@ -23,9 +23,9 @@ source $(dirname $0)/../vendor/github.com/knative/test-infra/scripts/library.sh
 CODEGEN_PKG=${CODEGEN_PKG:-$(cd ${REPO_ROOT_DIR}; ls -d -1 ./vendor/k8s.io/code-generator 2>/dev/null || echo ../../../k8s.io/code-generator)}
 
 # Generate based on annotations
-go generate ./pkg/... ./cmd/... ./contrib/github/pkg/... ./contrib/gcppubsub/pkg/... ./contrib/camel/pkg/... ./contrib/kafka/pkg/... ./contrib/awssqs/pkg/...
```

Comment:
I'm fine with this, however, I thought we had decided to call them importers instead of sources?

Reply:
ok :-) 

let me change that in a separate PR

I think there might be more to rename

this is just relocation - ok ?

Reply:
👍